### PR TITLE
Edits after #8644

### PR DESCRIPTION
**Pull request by bfallonf**
For: #8644 

cc @vikram-redhat @rlopez133 

@openshift/team-documentation PTAL

### CODE REVIEW

File: admin_guide/diagnostics_tool.adoc [hunk 1]
```diff
@@ -50,7 +49,7 @@ administrator, run:
 # oc adm diagnostics
 ----
 
-This runs all available diagnostics, skipping any that do not apply.
+This runs all available diagnostics and skips any that do not apply to the environment.
```
Comment (kalexand-rh) [inline admin_guide/diagnostics_tool.adoc]: I might s/this runs/this command starts

File: admin_guide/diagnostics_tool.adoc [hunk 2]
```diff
@@ -175,21 +173,18 @@ run.
 [[admin-guide-diagnostics-tool-client-environment]]
 == Running Diagnostics in a Client Environment
 
-You can access the diagnostics tool as an ordinary user, as a *cluster-admin*
-user, and can run on a host where {product-title} master or node servers are
-operating. The diagnostics attempt to use as much access as the user has
-available.
+The diagnostics tool runs using the level of permissions granted to the 
+account from which you run it. For example, as an ordinary user or a `cluster-admin` user.
```
Comment (kalexand-rh) [inline admin_guide/diagnostics_tool.adoc]: "For example, as an ordinary user or a `cluster-admin` user. " is a fragment. I think you can remove it without impacting the meaning of this section. Please do that or work it into the previous sentence.

File: admin_guide/diagnostics_tool.adoc [hunk 3]
```diff
@@ -228,14 +223,14 @@ using the provided *_health.yml_* playbook.
...
-required information. In some circumstances, additional system components, such
-as `docker` or networking configurations, can be altered if their current state
-differs from the configuration in the inventory file. You should run these
-health checks only if you do not expect the inventory file to make any changes
-to the existing cluster configuration. 
+you must run the playbooks against only Ansible-deployed clusters and using the
+same inventory file used for deployment. The changes consist of installing
+dependencies so that the checks can gather the required information. In some
+circumstances, additional system components, such as `docker` or networking
+configurations can be altered if their current state differs from the
```
Comment (kalexand-rh) [inline admin_guide/diagnostics_tool.adoc]: I'd s/can be altered/are changed

File: admin_guide/diagnostics_tool.adoc [hunk 4]
```diff
@@ -228,14 +223,14 @@ using the provided *_health.yml_* playbook.
...
+configuration in the inventory file. You should run these health checks only if
```
Comment (kalexand-rh) [inline admin_guide/diagnostics_tool.adoc]: s/You should run/Run

File: admin_guide/diagnostics_tool.adoc [hunk 5]
```diff
@@ -378,8 +373,8 @@ ifdef::openshift-origin[]
 endif::[]
 image via the Docker CLI.
 
-As a non-root user that has privileges to run containers specify the cluster's
-inventory file and the *_health.yml_* playbook:
+Run the following as a non-root user that has privileges to run containers and
+specify the cluster's inventory file and the *_health.yml_* playbook:
```
Comment (kalexand-rh) [inline admin_guide/diagnostics_tool.adoc]: Do we need to say "specify the cluster's inventory file and the *_health.yml_* playbook" here? Would adding an annotation to line 383 be better? If we do, I'd s/specify/specify both

File: admin_guide/diagnostics_tool.adoc [hunk 6]
```diff
@@ -422,9 +417,12 @@ becomes unable to access the public keys to allow remote login. To avoid
...
-default configuration and can be re-enabled with an environment variable by
-adding `-e ANSIBLE_HOST_KEY_CHECKING=True` to the `docker` command line.
+Mounting an entire *_.ssh_* directory can be helpful for:
+
+* Allowing you to use an SSH configuration to match keys with hosts or
+modify other connection parameters.
+* Allowing a user to provide a *_known_hosts_* file and have SSH validate host
+ keys. This is disabled by the default configuration and can be re-enabled with
+ an environment variable by adding `-e ANSIBLE_HOST_KEY_CHECKING=True` to the
+ `docker` command line.
```
Comment (kalexand-rh) [inline admin_guide/diagnostics_tool.adoc]: Is there an extra space here?

**Review by kalexand-rh — COMMENTED** [review]
I have a couple of suggestions and a request. This sounds good. :)

**Comment by bfallonf** [comment]
Thanks @kalexand-rh . Merging.

**Pull request merged**